root/etc/e-smith/events/actions/nethserver-directory-conf

```diff
@@ -33,6 +33,13 @@ if ( ! -f '/var/lib/ldap/cn.bdb' ) { # first configuration
     }
 }
 
+#
+# FIXME : slapd is not restarted by the event update, 
+# then systemd configuration for the service 
+# slapd-create-dhparam.service is not reloaded
+#
+system('/usr/bin/systemctl daemon-reload');
+
```

Comment:
The PR adds a .conf fragment and a .service definition: use RPM macros for systemd, according to Fedora RPM packaging guidelines.

```suggestion
```